src/Microsoft.AspNet.Diagnostics/project.json

```diff
@@ -3,6 +3,7 @@
     "description": "ASP.NET 5 Middleware for error handling, error pages, and diagnostics information.",
     "dependencies": {
         "Microsoft.AspNet.Diagnostics.Interfaces": "1.0.0-*",
+        "Microsoft.AspNet.Http.Extensions": "1.0.0-*",
```

Comment:
Don't need `Microsoft.AspNet.Http.Extensions` any more since the extension method have been moved out. Use `Microsoft.Framework.WebEncoders` directly instead.